### PR TITLE
Onboarding

### PR DESCRIPTION
**Pull request by ultimadj**
Initial instructions for Cheryl's self-service.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,7 +4,8 @@
   "scripts": {
     "build": "gulp deploy && gulp bundle-sw && gulp critical",
     "serve": "gulp deploy && gulp bundle-sw && gulp critical && firebase serve",
-    "deploy": "gulp deploy && gulp bundle-sw && gulp critical && firebase deploy"
+    "deploy": "gulp deploy && gulp bundle-sw && gulp critical && firebase deploy",
+    "tools": "npm i -g yarn firebase-tools"
```
Comment (markgoho) [inline package.json:8]: Awesome! Assume the user _doesn't_ have yarn yet!

File: README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+ ```
+ $ npm run tools
+ $ firebase login
+ $ yarn install
+ $ yarn serve
+ ```
+
+ ## Modify the Doula List
+ 1. Update: src/templates/partials/_doulas.pug
+ 2. Ensure any new profiles have a corresponding jpg and webp file in src/img/headshots (the filename needs to match the name key of the partial). Photos must be 200p wide by 280p tall (see: the-doulas.pug)
```
Comment (markgoho) [inline README.md]: Yeah......I do some pretty intense image editing and conversion to get the images in the right format. Conversion is done with the webp.js file, but the editing involves some art direction, cropping, and creating two sizes. There's a headshot, and then a headshot thumbnail.

I can produce these specs, but I'm not sure this process can/should be automated. Thoughts?

File: README.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# Doula Cooperative of Rochester
+ 
+ Site: https://www.doulacooperative.org
+
+ ## Getting Started
+ ```
+ $ npm run tools
+ $ firebase login
+ $ yarn install
```
Comment (markgoho) [inline README.md:9]: or just `yarn` 👍

File: README.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+ $ yarn serve
+ ```
+
+ ## Modify the Doula List
+ 1. Update: src/templates/partials/_doulas.pug
```
Comment (markgoho) [inline README.md]: `0. git checkout -b [doula-name]` where doula-name is just the name of the doula in [kebab-case](https://en.wikipedia.org/wiki/Letter_case#Special_case_styles)

File: README.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+ 2. Ensure any new profiles have a corresponding jpg and webp file in src/img/headshots (the filename needs to match the name key of the partial). Photos must be 200p wide by 280p tall (see: the-doulas.pug)
+ 3. Stop your server, then build and serve: `yarn serve`
+
```
Comment (markgoho) [inline README.md]: Maybe steps for submitting a PR? Not sure how detailed we want to get 😄

**Review by markgoho — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (ultimadj) [reply README.md]: Let's hold off on user-contributed images til there's a need.

File: README.md (hunk 3 above)
Comment (ultimadj) [reply README.md:9]: 👍

File: README.md (hunk 4 above)
Comment (ultimadj) [reply README.md]: I'm not planning to have Cheryl work with source control. The idea is:
1. I'll checkout the source and set her branch/working dir state
2. She'll make edits
3. I'll commit then pull from my pc
4. I'll test
5. I'll PR

File: README.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+ $ npm run tools
+ $ firebase login
+ $ yarn install
+ $ yarn serve
+ ```
+
+ ## Modify the Doula List
+ 1. Branch (recommendation: `<doula-name>`) 
+ 2. Update: src/templates/partials/_doulas.pug
+ 3. Ensure any new profiles have a corresponding jpg and webp file in src/img/headshots (the filename needs to match the name key of the partial). Photos must be 200p wide by 280p tall (see: the-doulas.pug) <- Contact @markgoho for new/updated images. They require special care & handling.
```
Comment (ultimadj) [inline README.md:16]: Left contact info for image updates

**Comment by ultimadj** [comment]
Updates performed. @markgoho

**Review by markgoho — APPROVED** [review]

**Pull request merged**